apps/rush-lib/src/cli/actions/AddAction.ts

```diff
@@ -128,8 +129,11 @@ export class AddAction extends BaseRushAction {
       throw new Error(`The package name "${packageName}" is not valid.`);
     }
 
-    if (version && version !== 'latest' && !semver.validRange(version) && !semver.valid(version)) {
-      throw new Error(`The SemVer specifier "${version}" is not valid.`);
+    if (version && version !== 'latest') {
+      const specifier: DependencySpecifier = new DependencySpecifier(packageName, version);
```

Comment:
Is this a new feature? If so, we should document any additional syntaxes supported by `rush add`.

I mention this because lately we've been adding a lot of new features, and sometimes the docs are still referring to the old behavior. This came up recently in https://github.com/microsoft/rushstack/issues/1951#issuecomment-647951548